packages/p2p/src/components/buy-sell/sell-table.jsx

```diff
@@ -0,0 +1,110 @@
+import React                  from 'react';
+import PropTypes              from 'prop-types';
+import { Loading }            from 'deriv-components';
+import {
+    RowComponent,
+    BuySellRowLoader }        from './row.jsx';
+import { InfiniteLoaderList } from '../table/infinite-loader-list.jsx';
+import { TableDimensions }    from '../table/table-dimensions.jsx';
+
+// TODO: replace with API response
+const mock_response = {
+    advertiser     : 'John Doe',
+    price          : 'IRD 2,000,000.00',
+    payment_method : 'Bank transfer',
+    type           : 'buy',
+    id             : 'buy_id',
+    country        : 'Indonesia',
+    currency       : 'IDR',
+    asset          : 'USD',
+    fix_price      : 12000,
+    amount         : 20,
+    min_transaction: 15000,
+    max_transaction: 240000,
+    advertiser_note: 'Whatsapp: +60182655318 please send to Maybank 239847238947 JOHN DOE',
+};
+
+const initial_data = [
+    { ...mock_response },
+    { ...mock_response },
+    { ...mock_response },
+    { ...mock_response },
+];
+
+const getMockData = () => {
+    return new Promise(resolve => {
+        setTimeout(function() {
+            resolve([
+                { ...mock_response }, { ...mock_response }, { ...mock_response }, { ...mock_response },
+            ]);
+        }, 300);
+    });
+};
+
+export class SellTable extends React.Component {
```

Comment:
can we use one table for buy and sell? it looks similar

Reply:
@negar-binary I think it's easier to separate them since they'll (probably) have different API endpoints and we might have to have special cases for either the buy or sell table - if not we can change this later